Skype/UCWA/audioVideoNegotiation_ref.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: audioVideoNegotiation
-description: A resource that provides an outgoing audioVideo call its media answers. This resource appears only in the event channel or in a response to a request. It will not appear in a request.
+description: A resource that provides an outgoing audioVideo call its media answers. This resource appears only in the event channel or in a response to a request.
```

Comment:
removed the last sentence intentionally? It will not appear in a request.

Reply:
Yes. The description was running a little long.